src/Microsoft.HttpRepl.Tests/Commands/SetSwaggerCommandTests.cs

```diff
@@ -377,12 +377,72 @@ public async Task ExecuteAsync_WithChildDirectories_SetsHttpStateSwaggerStructur
             Assert.Equal("Values", childDirectoryNames.First());
         }
 
-        private async Task<IDirectoryStructure> GetDirectoryStructure(string response, string parseResultSections, string baseAddress)
+        [Fact]
+        public async Task ExecuteAsync_WithAbsoluteUrl_SetsApiDefinition()
+        {
+            string response = @"{
+  ""swagger"": ""2.0"",
+  ""paths"": {
+    ""/api/Values"": {
+      ""post"": {
+        
+      }
+    }
+  }
+}";
+
+            string swaggerAddress = "http://localhost/swagger.json";
+            string baseAddress = "https://petstore.swagger.io/v2/";
```

Comment:
Can we use a junk value in the unit test?  So as to not confuse with an actual dependency on a real endpoint.